CONTRIBUTING.md

```diff
@@ -64,20 +70,29 @@ the repo to catch any issues locally.
 
 ### How to Receive Comments
 
-* If the PR is not ready for review, please put `[WIP]` in the title,
-  tag it as `work-in-progress`, or mark it as
-  [`draft`](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
-* Make sure CLA is signed and CI is clear.
+* If the PR is not ready for review, please put `[WIP]` in the title or mark it
+  as [`draft`](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
+* Make sure CLA is signed and all required CI checks are clear.
+* Submit small, focused PRs addressing a single concern/issue.
+* Make sure the PR title reflects the contribution.
+* Write a summary that helps understand the change.
+* Include usage examples in the summary, where applicable.
+* Include benchmarks (before/after) in the summary, for contributions that are
+  performance enhancements.
 
 ### How to Get PRs Merged
 
 A PR is considered to be **ready to merge** when:
 
-* It has received approval from Collaborators/Maintainers.
-* Major feedback is resolved.
+* It has received approval from
+  [Approvers](https://github.com/open-telemetry/community/blob/main/community-membership.md#approver).
+  /
+  [Maintainers](https://github.com/open-telemetry/community/blob/main/community-membership.md#maintainer).
+* Major feedbacks are resolved.
```

Comment:
we can decide on the number of approvals good enough to merge, depending on the size and other factors. Can be done separate of this PR.

Reply:
Yeah I'd like enforce 2 approval before merging given we have more approvers

Reply:
For now, maybe maintainers can decide to wait for 2 approvals before merging on a case-by-case basis, instead of requiring/enforcing it for every PRs (just to keep the momentum going!). Would this work?
(That is how many others SIGs are operating. But happy to revisit the idea for this SIG)

Reply:
Yeah we should call it out in documentation though. So that people will review the PR with 1 approval

Reply:
Do we want to? I guess maintainers/approvers are already doing so.... Writing something like this in official doc, without being too restrictive is hard 😄

Reply:
https://github.com/open-telemetry/opentelemetry-rust/pull/1354/commits/6a53393c0fa233e3928bef877dbe7a89d6e0babd Added some wording! @TommyCpp see if that makes sense!

Reply:
Thanks for addressing that